pillar/top.sls.in

```diff
@@ -1,3 +1,3 @@
-metalk8s-@VERSION@:
+metalk8s-§VERSION:
```

Comment:
Why?

Reply:
Replaced by `@@VERSION` now, which is more "portable" from a keyboard point of view.